app/main/forms/direct_award_forms.py

```diff
@@ -14,3 +15,26 @@ class CreateProjectForm(FlaskForm):
             Optional()
         ]
     )
+
+
+class DMRadioField(RadioField):
```

Comment:
It might be nice to add this to utils and stick a TODO in here to rip it out later when it's available through utils.

Not the end of the world and definitely don't hold off merging this to get it into utils and pull it in immediately.